server/websockets/websockets.js

```diff
@@ -131,13 +135,17 @@ async function websocketLogic(socket) {
   });
   socket.on('submitDrawing', async ({ gameRoom, teamRoom, drawing }) => {
```

Comment:
this is minor and it looks like I missed this, too, but this is no longer an asynchronous function. the compareImages function from resemblejs returned a promise, but the resemble...compareTo... functions are not.